src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/UpdateResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,223 @@
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave Edited",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 204
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 200
+              And the response body is
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave Edited",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+
+        @ignore
+        Scenario: Verify error handling updating a resource with invalid data
+            # The id value should be replaced with the resource created in the Background section
+             When a PUT request is made to "/ed-fi/absenceEventCategoryDescriptors/<id>" with
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave Edited",
+                        "namespace": "AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 400
+              And the response body is
+                  """
+                    {
+                        "detail": "Identifying values for the AbsenceEventCategoryDescriptor resource cannot be changed. Delete and recreate the resource item instead.",
+                        "type": "urn:ed-fi:api:bad-request:data",
+                        "title": "Data Validation Failed",
+                        "status": 400,
+                        "correlationId": null
+                    }
+                  """
+
+        @ignore
+        Scenario: Verify that response contains the updated resource ID and data
+            # The id value should be replaced with the resource created in the Background section
+             When a PUT request is made to "/ed-fi/absenceEventCategoryDescriptors/<id>" with
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave Edited",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 204
+                  | location |  |
+              And the response headers includes
+              #replace header <id> with the correct value
```

Comment:
Please use `{id}` for consistency.

The header response is not in JSON format, so it will be better to simply have:

```
location: ed-fi/absenceEventCategoryDescriptors/{id}
```

Reply:
Replaced in Resources Folder